calico-cloud/operations/ebpf/enabling-ebpf.mdx

```diff
@@ -235,8 +235,7 @@ not be disrupted, but they do not benefit from eBPF mode’s advantages.
 
 :::note
 
-the operator rolls out the change with a rolling update which means that some nodes will be in eBPF mode
-before others. This can disrupt the flow of traffic through node ports.
+The operator rolls out the change with a rolling update (non-disruptive) and then swiftly transitions all nodes to eBPF mode. However, it's inevitable that some nodes will enter eBPF mode before others.This can disrupt the flow of traffic through node ports.
```

Comment:
```suggestion
The operator rolls out the change with a rolling update (non-disruptive) and then swiftly transitions all nodes to eBPF mode. However, it's inevitable that some nodes will enter eBPF mode before others. This can disrupt the flow of traffic through node ports.
```

Needs space, here and in other files. 

Otherwise LGTM.